frontend/src/types.ts

```diff
@@ -1310,7 +1310,7 @@ export interface Experiment {
     name: string
     description: string
     feature_flag: string[]
-    filters: Partial<FilterType>
+    filters: Record<string, any>
```

Comment:
`filters` being a `FilterType` is indeed correct, isn't it?

Reply:
yeah but typescript hates it 😭 and I noticed we do that with a bunch of other keys in our interfaces that should really also be `FilterType`s